README.md

```
@@ -61,7 +70,24 @@ To set up the project on your local environment, follow these steps:
 
-6. **Lint the Code**
+6. **Lint and Format Code**
+
+-   Lint JavaScript:
+
     ```bash
     npm run lint
     ```
+
+-   Lint SCSS:
+    ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)